### PR TITLE
Disable okhttp redirects for ctxsvc

### PR DESCRIPTION
**Pull request by BenWu**

### CODE REVIEW

File: ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/SendRequestTest.java [hunk 1]
```diff
@@ -103,4 +103,35 @@ public void testSuccessfulSend() {
...
+    WithFailures.Result<PCollection<PubsubMessage>, PubsubMessage> result = pipeline
+        .apply(Create.of(input)).apply(SendRequest.of(pipeline.newProvider(true)));
+
+    PAssert.that(result.failures()).satisfies(messages -> {
+      Iterator<PubsubMessage> iterator = messages.iterator();
+      int messageCount = 0;
+
+      for (; iterator.hasNext(); iterator.next()) {
+        messageCount++;
+      }
```
Comment (jklukas) [inline ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/SendRequestTest.java:127]:
```suggestion
      int messageCount = Iterators.size(messages.iterator());
```

Slight preference for using a guava helper in this case.

**Review by jklukas — APPROVED** [review]
Looks good! One suggestion in the test.

File: ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/SendRequestTest.java (hunk 1 above)
Comment (BenWu) [reply ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/SendRequestTest.java:127]: Wow that's useful

Comment (jklukas) [reply ingestion-beam/src/test/java/com/mozilla/telemetry/contextualservices/SendRequestTest.java:127]: Guava has a bunch of helpers like this that can make code prettier. Don't expect any magic performance-wise, though. I'm pretty sure `Iterators.size()` is doing exactly the same thing you already have written out here.

**Pull request merged**